cfgov/alerts/post_sqs_messages.py

```diff
@@ -100,7 +100,11 @@ def post_to_chat(endpoint, username, message, issue_url):
     )
 
     for message in response.get('Messages', {}):
-        body = message.get('Body')
+        body = message.get('Body').replace(
+            '#', '# '  # Avoids erroneous Github issue link
+        ).replace(
+            '[Open]', ''  # We want to expand the link
+        )
```

Comment:
These are Very Good Changes, but I wonder if it's worth trying to make these changes in the [Jenkins SQS plugin](https://github.com/cfpb/jenkins-sqs-plugin) instead of here.

Reply:
Makes sense to modify the source, but in this case the true source is a Mattermost jenkins plugin that we forked & didn't alter too much. I think it'll be a little more straightforward to introduce the changes here.

Reply:
I'd make the change here to keep the jenkins plugin as agnostic as possible.